.circleci/config.yml

```diff
@@ -86,10 +87,22 @@ workflows:
       - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: instana/pipeline-feedback
           context: orb-publishing
+          checkout: false # We want to control the checkout because we want to inject the version in the scripts
           add-pr-comment: false
           fail-if-semver-not-indicated: true
           publish-version-tag: true
           ssh-fingerprints: 16:0b:f1:b0:af:47:67:d5:69:49:a4:d8:96:a8:42:8b
+          pre-steps:
+            - checkout
+            - run:
+                command: |
+                  CURRENT_VERSION=$(circleci orb info instana/pipeline-feedback | grep 'Latest' | awk '{ print $2 }' | awk -F '@' '{ print $2}' )
+                  SEMVER_INCREMENT=$(echo "${COMMIT_SUBJECT}" | sed -En 's/.*\[semver:(major|minor|patch|skip)\].*/\1/p')
+                  if [ "${SEMVER_INCREMENT}" = 'major' ]; then NEXT_VERSION=$(echo "${CURRENT_VERSION}" | awk 'BEGIN{FS=OFS="."} {$1+=1} 1'); fi
```

Comment:
Does CircleCi just increase the major|minor|patch number and not reset the following digits? According to https://github.com/CircleCI-Public/circleci-cli/blob/272a716f44303a55b8af6ae99de9d4548f48fa1a/api/api.go#L1117 this is the case, just wanted to ensure we end up with the same version number

Reply:
We need to set following digits to 0 according to https://github.com/Masterminds/semver/blob/master/version.go#L322